src/javascript/Shopper/ShopperRoutes.tsx

```diff
@@ -0,0 +1,13 @@
+import { Route, Routes } from 'react-router-dom'
+
+import ShopperLanding from './Public/ShopperLanding'
+
+const ShopperRoutes = () => {
+  return (
+    <Routes>
+      <Route index element={<ShopperLanding />} />
```

Comment:
I think what would make this more useable would be if there was a layout component using `<Outlet />`. You can do some really cool stuff here that would look like this:

```
<Route path="/" element={<LayoutComponent />}>
    <Route index element={<ShopperLanding />} />
</Route>
```

What do you think of this approach.